.github/workflows/detect-secrets.yaml

```
@@ -0,0 +1,19 @@
+name: Detect Secrets
+
+on: [push, pull_request]
+
+jobs:
+  detect-secrets:
+    runs-on: ubuntu-latest
+    container: python:latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Yelp's detect secrets
+      run: |
+        apt-get update && apt-get install -y jq
+        pip install yq
+        pip install detect-secrets==$(yq -r .[0].rev .pre-commit-config.yaml)
```

```suggestion
        pip install detect-secrets==$(yq -r .repos[0].rev .pre-commit-config.yaml)
```
Please try with this. `.pre-commit-config.yaml` format was updated/refactored based on recommendation from the `pre-commit` program. I think the build error is because the YAML parsing was not matching the new format.